src/App.js

```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import List from './components/List';
 
 const longestSerieComputed = (series) => {
-  
+  // Implementation goes here
```

Comment:
Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.